src/components/DayOverview/DayOverviewPdf.tsx

```diff
@@ -0,0 +1,204 @@
+"use client";
+
+import React from "react";
+import { Svg, Document, Page, Text, View, StyleSheet, Path } from "@react-pdf/renderer";
+import { faFlag, faSquare, IconDefinition } from "@fortawesome/free-solid-svg-icons";
+
+import { ParcelOfSpecificDateAndLocation } from "@/components/DayOverview/DayOverview";
+
+interface DayOverviewDateProps {
+    date: Date;
+}
+
+interface DayOverviewLocationProps {
+    location: string;
+}
+
+interface DayOverviewRowProps {
+    row: ParcelOfSpecificDateAndLocation;
+}
+
+interface DayOverviewPdfProps {
+    date: Date;
+    location: string;
+    data: ParcelOfSpecificDateAndLocation[];
+}
+
+interface DayOverviewCardProps {
+    date: Date;
+    location: string;
+    data: ParcelOfSpecificDateAndLocation[];
+}
+
+interface CustomSVGProps {
+    icon: IconDefinition;
```

Comment:
Could changing here help with later problem maybe?

Reply:
Don't think I can because I didn't define the interface `IconDefinition` myself. It is something from `fortawesome`